app/src/main/java/com/coderabbit/app/calculator/Car.kt

```
@@ -0,0 +1,11 @@
+package com.coderabbit.app.calculator
+
+class Car {
+    fun start(isElectric: Boolean): Boolean {
+        return if (isElectric) {
+            return true
+        } else {
+            throw Exception()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)